javascript/house-style.md

```diff
@@ -1006,14 +1006,18 @@ Client-side templating is typically to be avoided - e.g. in the example of an AJ
 
 #### Polyfills
 
+An example polyfill might add `requestAnimationFrame` for any grade-A browsers which don't support it.
+
 If you utilise a Polyfill, the JavaScript must meet these conditions:
 
   - It must _not_ override native browser functionality
   - It must match native browser functionality exactly (where possible)
   - It can be either third-party, or written in-house
-  - Must be installed via NPM
+  - Must be installed via NPM (or dynamically via [polyfill.io](https://polyfill.io/v2/docs/), at your discretion).
+
+Note that `polyfill.io` adds one additional blocking HTTP request, and if loading asynchronously one must be careful to manage loading order of scripts.  For these reasons it is preferred to bundle polyfills with your main application code.
```

Comment:
Is it missing a "the" before "loading order of scripts", as in:

"if loading asynchronously one must be careful to manage _the_ loading order of scripts"

Reply:
I don't think so, but with is more clear, so will add.